### PR TITLE
Fix/home

### PR DESCRIPTION
**Pull request by Bodyan322**

### CODE REVIEW

File: src/pages/Home/index.js [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
 
 class Home extends Component {
   componentDidMount() {
-    // eslint-disable-next-line react/destructuring-assignment
-    this.props.fetchUserDials();
+    const { fetchUserDials: defState } = this.props;
+    defState();
   }
 
   render() {
```
Comment (motifej) [inline src/pages/Home/index.js]: const select =( { user } ) => ({ userDials: user.userDials })
const mapDispatchToProps =   {
    fetchUserDials: fetchUserDialsAction,
  };

export default connect(select, mapDispatchToProps)(Home);

File: src/components/EnhancedTable/EnhancedTableHead.js [hunk 2]
```diff
@@ -57,9 +50,9 @@ class EnhancedTableHead extends Component {
           {rows.map(row => (
             <TableCell
               key={row.id}
-              align={row.numeric ? 'center' : 'left'}
+              align="center"
               sortDirection={orderBy === row.id ? order : false}
-              style={{ fontSize: '20px' }}>
+              style={{ fontSize: '20px', padding: 0 }}>
```
Comment (motifej) [inline src/components/EnhancedTable/EnhancedTableHead.js]: remove inline styles

File: src/pages/Home/index.js (hunk 1 above)
Comment (Bodyan322) [reply src/pages/Home/index.js]: have done

File: src/components/EnhancedTable/EnhancedTableHead.js (hunk 2 above)
Comment (Bodyan322) [reply src/components/EnhancedTable/EnhancedTableHead.js]: have done

File: src/pages/Home/index.js [hunk 3]
```diff
@@ -1,29 +1,46 @@
...
+    console.log('Home');
     const { data } = this.props;
+    console.log('data', data);
     return <EnhancedTable allUserData={data} />;
   }
 }
 
-const select = store => {
-  const data = store.user.userDials;
+// const select = ({ user, auth }) => {
```
Comment (motifej) [inline src/pages/Home/index.js]: why do you need this comments?

File: src/pages/Home/index.js [hunk 4]
```diff
@@ -1,29 +1,46 @@
 import React, { Component } from 'react';
 import { connect } from 'react-redux';
 import EnhancedTable from 'components/EnhancedTable/EnhancedTable';
-import { fetchUserDials } from 'actions/user';
+// import { fetchUserDials } from 'actions/user.actions';
```
Comment (motifej) [inline src/pages/Home/index.js]: what this comment do?

File: src/pages/Home/index.js [hunk 5]
```diff
@@ -1,29 +1,46 @@
...
+    // console.log('before');
+    const { fetchUserDials, uid } = this.props;
+    // console.log(defState, uid);
+    fetchUserDials(uid);
   }
 
   render() {
+    console.log('Home');
     const { data } = this.props;
+    console.log('data', data);
```
Comment (motifej) [inline src/pages/Home/index.js]: remove console if it final PR

Comment (motifej) [reply src/pages/Home/index.js]: use it just for local development

File: src/reducers/user.reducer.js [hunk 6]
```diff
@@ -26,9 +28,14 @@ export default combineEvents(
     [FETCH_USER]: state => ({ error: null }),
     [FETCH_USER_SUCCESS]: (state, { payload }) => ({ profile: payload }),
     [SAVE_PROFILE]: (state, { payload }) => ({ profile: payload }),
-    [FETCH_USER_CURRENCY_DIALS]: (state, { payload }) => ({
+    [FETCH_USER_CURRENCY_DIALS_REQUEST]: (state) => ({
```
Comment (motifej) [inline src/reducers/user.reducer.js]: what this block do ?
`[FETCH_USER_CURRENCY_DIALS_REQUEST]: (state) => ({})`
NOTHING!!!

File: src/sagas/userSaga/index.js [hunk 7]
```diff
@@ -18,9 +18,32 @@ export function* fetchUserSaga() {
...
+        const itemCopy = { ...item };
+        if (typeof itemCopy === 'object') {
+          itemCopy.id = key;
+        }
+        newListArr.push(itemCopy);
+      });
+  
+      return newListArr;
+    };
+    console.log(dataList);
```
Comment (motifej) [inline src/sagas/userSaga/index.js]: remove log!

File: src/sagas/userSaga/index.js [hunk 8]
```diff
@@ -18,9 +18,32 @@ export function* fetchUserSaga() {
...
+    yield put(fetchDialsSuccess(dataListArray()));
+  }catch(error){
+    console.log('error')
```
Comment (motifej) [inline src/sagas/userSaga/index.js]: remove log! use devtools!

File: src/pages/Home/index.js (hunk 5 above)
Comment (Bodyan322) [reply src/pages/Home/index.js]: i am so sorryy for this 
inattention.

Comment (Bodyan322) [reply src/pages/Home/index.js]: it is not final PR

File: src/reducers/user.reducer.js (hunk 6 above)
Comment (Bodyan322) [reply src/reducers/user.reducer.js]: I was going to add a spinner to this block

Comment (Bodyan322) [reply src/reducers/user.reducer.js]: i was going to add flag

**Pull request merged**